src/plugins/kibana_usage_collection/server/collectors/management/README.md

```diff
@@ -5,32 +5,38 @@ All user changed UI Settings are automatically collected.
 
 After adding a new setting you will be required to do the following steps:
 
-1. Update the [schema](./schema.ts) to include the setting name and schema type.
```

Comment:
I switched the order because updating the types first helps you with autocompletion when writing the schema 😇 

Also, I fixed a couple of lint warnings we had in this Markdown :)